pyproject.toml

```diff
@@ -30,12 +33,9 @@ pytest = "^8.4"
 pytest-black = "^0.3.7"
 pytest-cov = "^2.8"
 pytest-flake8 = "^1.0"
-pytest-mypy = "^0.8"
+pytest-mypy = "^1.0"
 pytest-pydocstyle = "^2.3"
-sphinx = "^5.0"
-sphinx-readable-theme = "^1.3.0"
```

Comment:
Moved to docs/requirements.txt as it is not needed from any poetry commands anymore.